views/js/review/component/qtiTestReviewComponent.js

```diff
@@ -40,7 +40,7 @@ define([
 
         const testRunnerConfig = {
             testDefinition: config.testDefinition || 'test-container',
-            serviceCallId: config.serviceCallId || 'review',
+            serviceCallId: config.testUri.resultId,
```

Comment:
Both `testUri` and `testUri.resultId` properties would always be passed through `config`?